docs/introduction.mdx

```diff
@@ -0,0 +1,63 @@
+---
+sidebar_position: 0
+---
+
+import QuickNavigation from "@site/src/components/QuickNavigation";
+import { items } from "@site/src/content/cardItems";
+
+# Getting started
```

Comment:
WDYT about changing title to `Introduction`. From Getting Started I would expect info how can I start implementing, not info about how it works.